### PR TITLE
Features/searching for movie and people

### PR DESCRIPTION
**Pull request by Siedemus**

### CODE REVIEW

File: src/Features/MoviesPage/index.js [hunk 1]
```diff
@@ -42,14 +48,16 @@ const MoviesPage = () => {
     history.replace(`${location.pathname}?${searchParams}`);
   }, [currentMoviePage]);
 
-  useEffect(() => {
```
Comment (DominikaBuczekSzkowron) [inline src/Features/MoviesPage/index.js:45]: why do you remove this? Genres are not working!
genres

File: src/Features/MoviesPage/index.js [hunk 2]
```diff
@@ -42,14 +48,16 @@ const MoviesPage = () => {
...
   return (
     <>
       <Container>
-        <Header>Popular movies</Header>
+        <Header>
+          {searchQuery !== ""
+            ? totalResults
+              ? `Search results for “${searchQuery}” (${totalResults})`
+              : `Sorry, there are no results for “${searchQuery}”`
+            : "Popular Movies"}
```
Comment (DominikaBuczekSzkowron) [inline src/Features/MoviesPage/index.js:59]: Do not forget about NoResultPage ;)

**Review by DominikaBuczekSzkowron — COMMENTED** [review]
Everyting is ok but add the NoResultPage and please correct displaying of genres. I think you remove it by mistake.

**Review by mileni67 — APPROVED** [review]

**Review by mileni67 — COMMENTED** [review]
You have conflict

**Review by DominikaBuczekSzkowron — APPROVED** [review]
well done

**Pull request merged**